tools/jlink-nuttx.c

```diff
@@ -424,6 +449,11 @@ struct symbols_s *RTOS_GetSymbols(void)
 
 uint32_t RTOS_GetNumThreads(void)
 {
+  if (!g_plugin_priv.ntcb)
```

Comment:
```suggestion
  if (g_plugin_priv.ntcb > 0)
```

Reply:
Ok, did this (though it must be ntcb == 0, not ntct > 0).

Reply:
Yes. I made a mistake. Thank you for taking care.